slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml

```diff
@@ -16,12 +22,25 @@ data:
   frontend-namespace: thoth-frontend-prod
   infra-namespace: thoth-infra-prod
   storage-bucket-name: zero-prod
+  logging-no-json: "0"
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ceph
+data:
   bucket-name: "thoth"
   bucket-prefix: "data"
   public-bucket-name: DH-PLAYPEN
   host: "https://s3-openshift-storage.apps.zero.massopen.cloud/"
   endpoint: "https://s3-openshift-storage.apps.zero.massopen.cloud/"
   host-url: "http://prometheus-portal-thoth-infra-prod.apps.zero.massopen.cloud"
```

Comment:
this should go to prometheus CM

Reply:
which one @pacospace ?

Reply:
`host-url: "http://prometheus-portal-thoth-infra-prod.apps.zero.massopen.cloud"`

Reply:
cool, we can open that in a different pr, that particular is not used in cronjob so it should fail it